1.html

```diff
@@ -0,0 +1,228 @@
+							target="_blank"
+							rel="noreferrer"
+							>Our Team</a
+						>
+					</li>
+					<li>
+						<a
+							href="https://links.ourtech.community"
+							target="_blank"
+							rel="noreferrer"
+							>All Links</a
+						>
+					</li>
+				</ul>
+			</nav>
+		</header>
+
+		<h1>OTC Talks #1</h1>
+
+		<section id="talks-details">
+
+			<h2>Table of Contents</h2>
+			
+			<ul id="talks-list">
+				<li>
+					<a href="#talk-1"
+						>Supercharge your deployments with WASM, by Rishit
+						Dagli</a
+					>
+				</li>
+			</ul>
+
+			<h2 id="when-where">Details</h2>
+
+			<ul>
+				<li><b>Date</b>: 24th December, 2022</li>
+				<li><b>Timing</b>: 10:30 PM IST</li>
+				<li>
+					<b>Where</b>: Online, held before
+					<a
+						href="https://catchup.ourtech.community/summary/111"
+						target="_blank"
+						rel="noreferrer"
+						>OTC CatchUp #111</a
+					>
+				</li>
+				<li><a href="/rules">Rules</a></li>
+				<li>
+					<a
+						href="https://ourtech.community/coc"
+						target="_blank"
+						rel="noreferrer"
+						>Code of Conduct</a
+					>
+				</li>
+			</ul>
+
+		</section>
+
+		<main id="main-content">
+			<section id="talk-1" class="talk-section">
+				<h2 class="talk-title">Supercharge your deployments with WASM</h2>
+				<p>
+					Speaker:
+					<a
+						href="https://twitter.com/rishit_dagli"
+						target="_blank"
+						rel="noreferrer"
+						>Rishit Dagli</a
+					>
+				</p>
+				<h3 id="about">Abstract</h3>
+				<p>
+					Machine Learning inference (though what we talk about can be
+					applied to other computationally intensive tasks) is often a
+					computationally intensive task and could greatly benefit
+					from the speed of WebAssembly. What if I told you, that you
+					could improve the performance of your application deployed
+					on Linux containers.
+				</p>
+				<p>Enter Wasm and Kubernetes.</p>
+				<p>
+					This talk starts off by introducing the audience to
+					WebAssembly (Wasm) and how they could make use of the speed
+					and security among others of Wasm for their deployments.
+					Another problem we face is that the standard WebAssembly
+					provides very limited access to the native OS and hardware,
+					such as multi-core CPUs, GPU, or TPUS which is not ideal for
+					the kind of systems we target. The talk also shows how one
+					could use the WebAssembly System Interface (WASI) to get
+					security, portability, and native speed for Machine Learning
+					models. The talk then slides into the recent (still preview)
+					support that Azure has launched for WASI node pools to run
+					such a system while also using Krustlet to run Wasm on
+					Kubernetes. To top it off this talk ends with a demo of
+					doing so on Azure.
+				</p>
+				<p>
+					Feel free to
+					<a
+						href="https://links.ourtech.community"
+						target="_blank"
+						rel="noreferrer"
+						>contact us</a
+					>.
+				</p>
+
+			<h3 id="relevant-links">Relevant Links</h3>
```

Comment:
I think this needs to be

```html
<h3 id="relevant-links-1">Relevant Links</h3>
```

because there can be multiple talks.

Please check for any other instances.

Reply:
Fixed.